### PR TITLE
fix: function names

### PR DESCRIPTION
**Pull request by 38elements**
I think this make it easy to read and `git blame`.

### CODE REVIEW

File: packages/create-instance/add-slots.js [hunk 1]
```diff
@@ -21,23 +23,54 @@ function createElementFromAdvancedString (slotValue, vm) {
...
+  }
+  if (typeof _window === 'undefined') {
+    throwError('the slots string option does not support strings in server-test-uitls.')
+  }
+  if (_window.navigator.userAgent.match(/PhantomJS/i)) {
+    throwError('the slots option does not support strings in PhantomJS. Please use Puppeteer, or pass a component.')
+  }
+}
+
+function addSlotToVm (vm: Component, slotName: string, slotValue: Component | string | Array<Component> | Array<string>): void {
```
Comment (eddyerburgh) [inline packages/create-instance/add-slots.js]: Could you move the slotValue into its own type:

`type SlotValue = Component | string | Array<Component> | Array<string>`

**Comment by 38elements** [comment]
I think a person who do not understand specifications and codes can not add large cosmetic change appropriately.

File: packages/create-instance/add-slots.js (hunk 1 above)
Comment (38elements) [reply packages/create-instance/add-slots.js]: Thank you for pointng out.
I added a type.

**Comment by eddyerburgh** [comment]
I've skipped the failing tests—https://github.com/vuejs/vue-test-utils/pull/581

File: packages/create-instance/add-slots.js [hunk 2]
```diff
@@ -21,23 +24,54 @@ function createElementFromAdvancedString (slotValue, vm) {
...
+  }
+  if (window.navigator.userAgent.match(/PhantomJS/i)) {
+    throwError('the slots option does not support strings in PhantomJS. Please use Puppeteer, or pass a component.')
+  }
+}
+
+function addSlotToVm (vm: Component, slotName: string, slotValue: SlotValue): void {
+  let elem
   if (typeof slotValue === 'string') {
-    slotValue = slotValue.trim()
```
Comment (38elements) [inline packages/create-instance/add-slots.js:26]: When `slotValue` is `'       foo      '`, this line make a bug.
This process introduced at #556 .
I added a test for spaces.

File: test/specs/mounting-options/slots.spec.js [hunk 3]
```diff
@@ -109,10 +109,13 @@ describeWithMountingMethods('options.slots', (mountingMethod) => {
       const wrapper5 = mountingMethod(ComponentWithSlots, { slots: { default: '1{{ foo }}2' }})
       expect(wrapper5.find('main').html()).to.equal('<main>1bar2</main>')
       wrapper5.trigger('keydown')
+      // expect(wrapper5.find('main').html()).to.equal('<main>1BAR2</main>')
```
Comment (38elements) [inline test/specs/mounting-options/slots.spec.js]: @eddyerburgh 
This line does not work.
Would you please check below links?

* https://github.com/vuejs/vue-test-utils/pull/274#issuecomment-352262141

* https://github.com/vuejs/vue-test-utils/pull/274/commits/3b99090a6917b7861096b2903fc9ec7811607464#diff-dbfd1670fb2dd50be051adb6643c8c24

**Pull request merged**